plat/kvm/arm/link64.lds.S

```diff
@@ -29,6 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+#define UK_USE_SECTION_SEGMENTS 1
```

Comment:
Hi, is there a reason why you did not do this define inside the platforms' `Makefile.uk` instead? As in a `-DUK_USE_SECTION_SEGMENTS`. I believe this way we could avoid changing all the ld scripts and it would look more transparent. Unless I am missing something.